src/js/components/Select/SelectContainer.js

```diff
@@ -42,6 +42,34 @@ const ClearButton = forwardRef(
     const align = position !== 'bottom' ? 'start' : 'center';
     const buttonLabel = label || `Clear ${name || 'selection'}`;
     const { passThemeFlag } = useThemeValue();
+    const buttonKind = theme.select.clear?.button;
+
+    if (buttonKind) {
+      // New structure when `theme.select.clear.button` is defined
+      const containerProps = theme.select.clear?.container || {};
+      const textProps = theme.select.clear?.text || {};
+
+      return (
+        <Box flex="grow" {...containerProps}>
+          <StyledButton
+            a11yTitle={`${buttonLabel}. Or, press ${
+              position === 'bottom' ? 'shift tab' : 'down arrow'
+            } to move to select options`}
+            fill="horizontal"
+            justify="start"
```

Comment:
Why do we add justify="start" here?

Reply:
good catch Ill remove this. I was testing something and forgot to remove

Reply:
done